Rdmp.Core/Logging/LogManager.cs

```diff
@@ -236,9 +240,10 @@ public IDataLoadInfo CreateDataLoadInfo(string dataLoadTaskName, string packageN
     public void CreateNewLoggingTask(int id, string dataSetID)
     {
         using var conn = Server.GetConnection();
+        var sh = Server.GetQuerySyntaxHelper();
         conn.Open();
         var sql =
-            $"INSERT INTO DataLoadTask (ID, description, name, createTime, userAccount, statusID, isTest, dataSetID) VALUES ({id}, @dataSetID, @dataSetID, @date, @username, 1, 0, @dataSetID)";
+            $"INSERT INTO {sh.EnsureWrapped("DataLoadTask")} ({sh.EnsureWrapped("ID")}, description, name, {sh.EnsureWrapped("createTime")}, {sh.EnsureWrapped("userAccount")}, {sh.EnsureWrapped("statusID")}, {sh.EnsureWrapped("isTest")}, {sh.EnsureWrapped("dataSetID")}) VALUES ({id}, @dataSetID, @dataSetID, @date, @username, 1, false, @dataSetID)";
```

Comment:
Unsure whether changing 0 -> false here is correct, but postgres complains about int/bool column type mismatches otherwise. Will this work for other databases?

Reply:
This won't work for MSSQL as it uses the bit type for Booleans. We could add bool support into fansisql  so that we could do something like "Server.GetQuerySyntaxHelper().False" and have it return 0/false/etc as appropriate.
Happy to add this, but it'll take a few days to trickle through

Reply:
@rkm if you pull in the latest changes from develop you should be able to use "sh.False" to make this change db agnostic

Reply:
@JFriel for the longer term, we should probably update LogManager to use FAnsiSql instead of bare SQL - it's old code that predates FAnsi's existence IIRC, but would benefit from that in maintanability IMO?